### PR TITLE
Backend endpoints created & frontend pulls announcements to display on site

### PR DESCRIPTION
**Pull request by youzhentsai**
announcements appear on site, created backend endpoints to create/edit/delete announcement

### CODE REVIEW

File: backend/queries.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+const { userName } = require("./config");
+const { userPassword } = require("./config");
```
Comment (abaruah117) [inline backend/queries.js]: you can combine these 2 into one line

File: backend/queries.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+const Pool = require("pg").Pool;
+const pool = new Pool({
+  user: userName,
+  host: "localhost",
+  database: "ethical",
```
Comment (abaruah117) [inline backend/queries.js]: move host/database/port into config too

File: backend/queries.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      res
+        .status(201)
+        .send(`Announcement created with ID: ${results.rows[0].id}`);
+    }
+  );
+};
+
+const editAnnouncement = (req, res) => {
+  const id = parseInt(req.params.id);
+  console.log(id);
```
Comment (abaruah117) [inline backend/queries.js]: remove console.logs

File: backend/queries.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  const { title, info } = req.body;
+
+  pool.query(
+    "UPDATE announcements SET title = $1, info = $2 WHERE id = $3",
+    [title, info, id],
+    (error, results) => {
+      if (error) {
+        throw error;
+      }
+      console.log(results);
```
Comment (abaruah117) [inline backend/queries.js]: remove

File: backend/queries.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+        throw error;
+      }
+      res.status(200).json(results.rows);
+    }
+  );
+};
+
+const createAnnouncement = (req, res) => {
+  const { title, info } = req.body;
+  console.log([title, info]);
```
Comment (abaruah117) [inline backend/queries.js]: remove

**Review by abaruah117 — CHANGES_REQUESTED** [review]
looks good I think

File: src/components/Announcements.js [hunk 6]
```diff
@@ -1,28 +1,22 @@
...
   }
-];
 
-class Announcements extends React.Component {
   render() {
     return (
       <div className="Announcements-section">
-        {postValues.map(item => (
+        {this.state.postValues.map(item => (
           <Post key={item.id} {...item} />
```
Comment (trev4ev) [inline src/components/Announcements.js:20]: <3 this

File: backend/config.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+const userId = 3376787;
```
Comment (trev4ev) [inline backend/config.js]: update .gitignore so this file isn't committed

**Review by trev4ev — CHANGES_REQUESTED** [review]
Looks good to me! just need to update .gitignore. We can work through merging this at worksession since we will need to erase some commits.

File: .vscode/launch.json [hunk 8]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (abaruah117) [inline .vscode/launch.json]: remove this pls

File: backend/config.js (hunk 7 above)
Comment (abaruah117) [reply backend/config.js]: +1

**Review by abaruah117 — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 8 above)
Comment (abaruah117) [inline .vscode/launch.json]: this file is still here LOL

**Review by abaruah117 — CHANGES_REQUESTED** [review]
please verify that you address changes before requesting again!

File: backend/app.js [hunk 9]
```diff
@@ -1,21 +1,29 @@
 const express = require("express");
 const app = express();
+const bodyParser = require("body-parser");
 const port = 3001;
+const db = require("./queries");
```
Comment (trev4ev) [inline backend/app.js]: sorry for not catching this earlier, but can this be renamed to show that it's specifically for announcements?

**Review by trev4ev — CHANGES_REQUESTED** [review]
One last request for me! :)

**Review by trev4ev — APPROVED** [review]
Looks good to me!

**Review by abaruah117 — APPROVED** [review]

**Pull request merged**